test/e2e/persistent_volumes.go

```diff
@@ -119,48 +117,47 @@ func deletePersistentVolumeClaim(c clientset.Interface, pvcName string, ns strin
 	}
 }
 
-// Delete the PVC and wait for the PV to become Available again. Validate that the PV
+// Delete the PVC and wait for the PV to enter its expected phase. Validate that the PV
 // has recycled (assumption here about reclaimPolicy). Caller tells this func which
```

Comment:
I think the sentence about validating that the pv has recycled is a bit misleading since in most cases the pv will not be recycled. I think it's ok to just say _and wait for the PV to enter its expected phase_.